README.md

```diff
@@ -38,7 +38,7 @@ You've now installed the code base.
 Enter the following command into terminal from the same directory you ran the above commands:
 
 ```
-$ bash start_docker.sh
+bash start_docker.sh
```

Comment:
If we're going to remove the $ from this command, we should do the same for the ones above in the Readme